src/main/scala/com/charlesahunt/scalapb/ScalaPB.scala

```diff
@@ -112,7 +113,7 @@ object ProtocPlugin extends LazyLogging {
           pluginFrontend = protocbridge.frontend.PluginFrontend.newInstance
       )
     } catch { case e: Exception =>
-      throw new RuntimeException("Error occurred while compiling protobuf files: %s" format(e.getMessage), e)
+      throw new RuntimeException("Error occurred while compiling protobuf files: %s" format e.getMessage, e)
```

Comment:
Just my 5 cent here, but just because you can remove braces and dots in Scala doesnt mean you have to. This is not a Scala-DSL and since @CharlesAHunt defined the style you should stick to it.